cmd/vic-machine/create/create.go

```diff
@@ -454,7 +453,7 @@ func (c *Create) Flags() []cli.Flag {
 func (c *Create) processParams() error {
 	defer trace.End(trace.Begin(""))
 
-	if err := c.HasCredentials(); err != nil {
+	if err := c.HasOpCredentials(c.Force); err != nil {
```

Comment:
This method will check operation credentials, and admin credentials, why not reuse the name HasCredentials?